src/app/state/coordination.js

```diff
@@ -6,6 +6,9 @@ import { CoordinationType, Component } from '../constants';
  * auto initialization strategy.
  */
 export const DEFAULT_COORDINATION_VALUES = {
+  [CoordinationType.OBS_TYPE]: 'cell',
+  [CoordinationType.FEATURE_TYPE]: 'gene',
+  [CoordinationType.FEATURE_VALUE_TYPE]: 'expression',
```

Comment:
at the minimum, we should note that these values are totally arbitrary and only give the view config readability.  it might even make sense to make these empty strings....not sure.